README.md

```diff
@@ -1,4 +1,224 @@
+
+```
+nodeTLSSecret: cockroachdb.node
+clientTLSSecret: cockroachdb.client.root
+``` -->
+
+### Set resource requests and limits
+
+Before deploying, we recommend explicitly allocating CPU and memory to CockroachDB in the Kubernetes pods. 
+
+Enable the commented-out lines in the `resources.requests` object and substitute values appropriate for your workload. Note that `requests` and `limits` should have identical values.
+
+```
+resources:
+  requests:
+    cpu: "4"
+    memory: "16Gi"
+    storage: 60Gi
+  limits:
+    cpu: "4"
+    memory: "16Gi"
+```
+
+### Apply the StatefulSet configuration
+
+Apply `example.yaml`:
+
+```
+kubectl create -f example.yaml
+```
+
+Check that the pods were created:
+
+```
+kubectl get pods
+```
+
+```
+NAME                                  READY   STATUS    RESTARTS   AGE
+cockroach-operator-6f7b86ffc4-9t9zb   1/1     Running   0          3m22s
+cockroachdb-0                         1/1     Running   0          2m31s
+cockroachdb-1                         1/1     Running   0          102s
+cockroachdb-2                         1/1     Running   0          46s
+```
+
+Each pod should have `READY` status soon after being created.
+
+## Access the SQL shell
+
+Get a shell into one of the pods and start the CockroachDB built-in SQL client:
+
+```
+kubectl exec -it cockroachdb-2 -- ./cockroach sql --certs-dir=/cockroach-certs
+```
+
+If you want to [access the Admin UI](#access-the-admin-ui), create a SQL user with a password while you're here:
+
+```
+CREATE USER roach WITH PASSWORD 'Q7gc8rEdS';
+```
+
+```
+\q
+```
+
+## Access the Admin UI
+
+To access the cluster's [Admin UI](https://www.cockroachlabs.com/docs/v20.2/admin-ui-overview), port-forward from your local machine to the `cockroachdb-public` service:
+
+```
+kubectl port-forward service/cockroachdb-public 8080
+```
+
+Access the Admin UI at `https://localhost:8080`. Note that [certain Admin UI pages](https://www.cockroachlabs.com/docs/v20.2/admin-ui-overview#admin-ui-access) require `admin` access, which you can grant to the SQL user you created earlier:
+
+```
+GRANT admin TO roach;
+```
+
+### Scale the cluster
+
+To scale the cluster up and down, edit `nodes` in `example.yaml`:
+
+```
+nodes: 4
+```
+
+Do **not** scale down to fewer than 3 nodes. This is considered an anti-pattern on CockroachDB and will cause errors.
+
+> Note that you must scale by updating the `nodes` value in the Operator configuration. Using `kubectl scale statefulset <cluster-name> --replicas=4` will result in new pods immediately being terminated.
+
+Apply the new configuration:
+
+```
+kubectl apply -f example.yaml
+```
+
+### Upgrade the cluster
+
+Perform a rolling upgrade by changing the image name in `example.yaml`:
+
+```
+image:
+  name: cockroachdb/cockroach:v20.2.0
+```
+
+Apply the new configuration:
+
+```
+kubectl apply -f example.yaml
+```
+
+## Stop the cluster
```

Comment:
We need to have specific English around a cockroach cluster vs a Kubernetes cluster.  This may sound like the user is deleting or stopping the k8s cluster.